docs/source/user_guide/using_datasets.rst

```diff
@@ -4501,96 +4553,26 @@ to generate orthographic projection images of each point cloud:
 
     session = fo.launch_app(dataset)
 
-The above method populates an |OrthographicProjectionMetadata| field on each
-sample that contains the path to its projection image and the necessary to
-properly :ref:`visualize it in the App <app-3d-orthographic-projections>`.
-
 .. note::
 
-    Refer to the
-    :func:`compute_orthographic_projection_images() <fiftyone.utils.utils3d.compute_orthographic_projection_images>`
-    documentation for available parameters to customize the projections.
-
-.. _point-cloud-datasets:
-
-Point cloud datasets
---------------------
-
-.. note::
-
-    Deprecation notice:
-
-    The `point-cloud` media type has been deprecated in favor of the
-    `3d` media type. While we'll keep supporting the `point-cloud` media type
-    for backward compatibility, we recommend using the `3d` media type for new
-    datasets.
-
-Any |Sample| whose `filepath` is a
-`PCD file <https://pointclouds.org/documentation/tutorials/pcd_file_format.html>`_
-with extension `.pcd` is recognized as a point cloud sample, and datasets
-composed of point cloud samples have media type `point-cloud`:
+    You must set `flag_for_projection=True` on any
+    :ref:`point clouds <3d-point-clouds>` that you wish to be projected.
```

Comment:
Same comment as above. Might want to change the imperative from "must" to something lighter

Reply:
👍